variables.tf

```diff
@@ -62,6 +62,12 @@ variable "root_block_device_volume_size" {
   description = "The volume size (in GiB) to provision for the root block device. It cannot be smaller than the AMI it refers to."
 }
 
+variable "root_block_device_volume_type" {
+  type        = string
+  default     = "gp3"
+  description = "The volume type."
```

Comment:
Can you please list the valid values here? Or if this module requires 0.13+ then please include a `validation` block to ensure the user is only passing in accepted values.

Reply:
Yes, done in PR #77.

Closing this PR because it included incomplete edits from a different feature.